acoustid/api/v2/__init__.py

```
@@ -67,17 +67,17 @@
 MAX_RESULTS_PER_FINGERPRINT_QUERY = 10
 
 
-def iter_args_suffixes(args, *prefixes):
-    results = set()
+def iter_args_suffixes(args, *prefixes) -> Iterable[str]:
+    results: set[int] = set()
     for name in args.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)